src/attributes/classes.js

```diff
@@ -86,7 +84,7 @@ jQuery.fn.extend({
 					}
 
 					// Only assign if different to avoid unneeded rendering.
-					finalValue = value ? jQuery.trim( cur ) : "";
```

Comment:
No, no it's not for save a function call, it was a specific behaviour, if the `value` of the user is `undefined` we remove set an empty class attribute. It is really more simple to read the specific case of `.removeClass()` now.